src/components/data-processing/ConfigureEmbedding/ConfigureEmbedding.jsx

```diff
@@ -65,14 +64,15 @@ const ConfigureEmbedding = (props) => {
       plotType: 'embeddingPreviewMitochondrialContent',
       plot: (config) => (<MitochondrialContentPlot experimentId={experimentId} config={config} plotUuid='embeddingPreviewMitochondrialContent' />),
```

Comment:
I know it is not part of your PR, but I will say it here so it doesn't get lost -- similarly in the other plots, looks like this `plotUuid` is actually not used. We should either use it or stop passing it as props -- it isn't a good engineering practice to keep things that are unused. So whatever you end up doing for `doubletScores` (my previous comment about using `plotUuid` from props instead of plot name), just make sure that you apply it to the other plots so that it is consistent.